tests/.eslintrc.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   env: {
     'embertest': true
+  },
+  globals: {
+    'expect': true
```

Comment:
you can probably just use the `mocha` env here which also includes `describe`, `it`, ...